jsk_robot_common/jsk_robot_startup/scripts/shutdown.py

```diff
@@ -51,6 +56,20 @@ def __init__(self):
             '~shutdown_command', '/sbin/shutdown -h now')
         self.reboot_command = rospy.get_param(
             '~reboot_command', '/sbin/shutdown -r now')
+        self.volume = rospy.get_param('~volume', 1.0)
+
+    def speak(self, text, lang='jp'):
+        if self.client.wait_for_server(timeout=rospy.Duration(1.0)):
```

Comment:
How about parameterizing the timeout value?

Reply:
Thanks for your advice. I added parameters.